src/main/Java/com/lonesurvivor/Main.java

```diff
@@ -14,9 +15,22 @@
 import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) throws IOException, ParseException, LineUnavailableException {
-        GameEngine game = new GameEngine();//create and init start of game
-        GuiStartPage guiStartPage = new GuiStartPage(game);//create init and start UI page
+    public static void main(String[] args) {
+        GameEngine game = null;//create and init start of game
+        try {
+            game = new GameEngine();
+        } catch (IOException | ParseException e) {
+            e.printStackTrace();
+        }
+        try {
+            try {
```

Comment:
this will work however nesting try/catch blocks within each other impacts readability and makes it hard to see which block catches which exception